identity/engines/pkcs11/engine_linux_test.go

```diff
@@ -16,9 +16,17 @@
 
 package pkcs11
 
-import (
-	_ "net/url"
-)
+import "os"
 
-const pkcs11Lib = "/usr/lib/softhsm/libsofthsm2.so"
+func getPkcs11Lib() (string) {
+	libFromEnv := os.Getenv(softHsmEnvVar)
+
+	if len(libFromEnv) > 0 {
+		log.Infof("LIB FROM ENV")
```

Comment:
Do you intend to log the value of the env variable here? If not, this should probably be an `Info` vs `Infof`.
Also, I suggest logging the variable name so somebody doesn't need to look at the code to know what they need to set/change. e.g.:

    log.Infof("Using pkcs11 module %s, set via %s env var", libFromEnv, softHsmEnvVar)